_posts/2024-05-23-pyos-call-for-editors.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title: "Volunteers needed: pyOpenSci Editorial Board"
+excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-call-for-editors-may-2024.html
+header:
+  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## Join the pyOpenSci Editorial Board
```

Comment:
can you please link to this page - https://www.pyopensci.org/software-peer-review/how-to/editors-guide.html somewhere towards the top of this blog?

Reply:
fixed!